.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release a new version
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    if: "!github.event.release.prerelease"
```

Comment:
Don't you want to release RC as pre-release?

Reply:
Yep that would be great. For the moment I deactivate the Azure deployment in case of prerelease to be sure that, in the future when we'll work on RC as prerelease, there is no risk to trigger a real release